src/osdep/linux/sysinfo.cpp

```diff
@@ -16,13 +16,7 @@
 // small hack to access environ variable conforming to the GNU way
 #define environ __environ
 
-#define STAT_FILE    "/proc/stat"
-static int stat_fd = -1;
-#define UPTIME_FILE  "/proc/uptime"
-static int uptime_fd = -1;
-
 long smp_num_cpus;     /* number of CPUs */
-static char buf[1024];
```

Comment:
Isn't `buf` used in `FILE_TO_BUF(filename, fd)` a few lines below?

Reply:
Which can also be removed :-)